<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaDatos</class>
 <widget class="QWidget" name="VentanaDatos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingresar Datos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_nombre">
     <property name="text">
      <string>Nombre completo:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_nombre"/>
   </item>
   <item>
    <widget class="QLabel" name="label_cedula">
     <property name="text">
      <string>Número de cédula:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_cedula"/>
   </item>
   <item>
    <widget class="QPushButton" name="boton_enviar">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
